docs/Overview/GuideOverview-template.markdown

```diff
@@ -86,16 +77,17 @@ territory in which the app is made available.
 
 ## Logging Errors and Warnings
 
-The ResearchKit framework supports four log levels, controlled by four preprocessor macros and their corresponding *`NSLog()`-like* logging macros:
+The *ResearchKit framework* supports four log levels, controlled by four preprocessor macros and their corresponding *`NSLog()`-like* logging macros:
 * `ORK_LOG_LEVEL_NONE`
 * `ORK_LOG_LEVEL_DEBUG`, `ORK_Log_Debug()`
 * `ORK_LOG_LEVEL_WARNING`, `ORK_Log_Warning()`
 * `ORK_LOG_LEVEL_ERROR`, `ORK_Log_Error()`
 
-Setting the ResearchKit's framework `ORK_LOG_LEVEL_NONE` macro to `1` completely silences all ResearchKit logs, overriding any other specified log level. Setting `ORK_LOG_LEVEL_DEBUG`, `ORK_LOG_LEVEL_WARNING`, or `ORK_LOG_LEVEL_ERROR` to `1` enables logging at that level and at those of higher seriousness.
+Setting the *ResearchKit framework* `ORK_LOG_LEVEL_NONE` macro to `1` completely silences all ResearchKit logs, overriding any other specified log level. Setting `ORK_LOG_LEVEL_DEBUG`, `ORK_LOG_LEVEL_WARNING`, or `ORK_LOG_LEVEL_ERROR` to `1` enables logging at that level and at those of higher seriousness.
 
 If you do not explicitly set a log level, `ORK_LOG_LEVEL_WARNING=1` is used by default.
 
 You have to set any of these preprocessor macros in your ResearchKit subproject, not in your main project. Within *Xcode*, you can do so by setting any of them in the `Preprocessor Macros` list on the `Build Settings` of your `ResearchKit` framework target.
 
 See these resources if you are using ResearchKit through CocoaPods and need to change the log level: [[1]](http://stackoverflow.com/a/30038120/269753) [[2]](http://www.mokacoding.com/blog/cocoapods-and-custom-build-configurations/).
+available.
```

Comment:
Nope! Removed. I also took the change to fix some minor *headerdoc* warnings that popped up when regenerating the documentation.